src/autorest-core/resources/default-configuration.md

```diff
@@ -93,9 +93,33 @@ pipeline:
   swagger-document/semantic-validator:
     input: transform
     scope: semantic-validator
+```
+
+##### Azure Validator
+
+``` yaml
+pipeline:
   swagger-document/azure-validator:
     input: transform
-    scope: azure-validator
+    scope: azure-validator-composed
+  swagger-document/individual/azure-validator:
+    input: individual/transform
+    scope: azure-validator-individual
+```
+
+Activate `azure-validator` when setting `azure-arm`!?
+
+``` yaml $(azure-arm)
+azure-validator: true
+```
+
+``` yaml $(azure-validator)
+azure-validator-composed:
+  merge-state: composed
+  openapi-type: arm
```

Comment:
not sure I follow this, defining multiple settings under the same scope is nested? What does setting default on the top mean?